### PR TITLE
CQ-4351074 & CQ-4351274 - Value commit script tests for fields

### PR DESCRIPTION
**Pull request by TalmizAhmed**
## Description
Tests for set value and clear value for the following fields in core components:
- Textinput
- NumberInput
- dropdown
- checkbox
- radiobutton
- datepicker

Clear value for Radio button wasn't working([CQ-4351274](https://jira.corp.adobe.com/browse/CQ-4351274)), so fixed that as well in this PR.
Also included a few validation exp. tests that were missed.

## Related Issue
https://jira.corp.adobe.com/browse/CQ-4351074
https://jira.corp.adobe.com/browse/CQ-4351274

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/clientlibs/site/js/radiobuttonview.js [hunk 1]
```diff
@@ -108,19 +108,17 @@
...
-                        widget.setAttribute(FormView.Constants.ARIA_CHECKED, true);
-                    } else {
-                        widget.checked = false;
-                        widget.removeAttribute(FormView.Constants.HTML_ATTRS.CHECKED);
-                        widget.setAttribute(FormView.Constants.ARIA_CHECKED, false);
-                    }
-                }, this)
-            }
+            this.widget.forEach(widget => {
+                if (modelValue && widget.value != null && modelValue.toString() == widget.value.toString()) {
```
Comment (rismehta) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/clientlibs/site/js/radiobuttonview.js:112]: Why have you moved this check ?

**Review by rismehta — CHANGES_REQUESTED** [review]
minor comment

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/clientlibs/site/js/radiobuttonview.js (hunk 1 above)
Comment (nit23uec) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/clientlibs/site/js/radiobuttonview.js:112]: if model value is different from widget value, how would the widget get updated to the new value?

**Comment by rismehta** [comment]
Incorporate comments, closing the PR

**Pull request merged**